frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -448,184 +468,194 @@ const DeviceUserPage = ({
-                      endUsers={host.end_users ?? []}
-                      enableAddEndUser={false}
-                      disableFullNameTooltip
-                      disableGroupsTooltip
-                    />
-                  )}
-                  {isAppleHost && !!deviceCertificates?.certificates.length && (
-                    <CertificatesCard
-                      className={fullWidthCardClass}
-                      isMyDevicePage
-                      data={deviceCertificates}
-                      isError={isErrorDeviceCertificates}
-                      page={certificatePage}
-                      pageSize={DEFAULT_CERTIFICATES_PAGE_SIZE}
-                      sortHeader={sortCerts.order_key}
-                      sortDirection={sortCerts.order_direction}
-                      hostPlatform={host.platform}
-                      onSelectCertificate={onSelectCertificate}
-                      onNextPage={() => setCertificatePage(certificatePage + 1)}
-                      onPreviousPage={() =>
-                        setCertificatePage(certificatePage - 1)
-                      }
-                      onSortChange={setSortCerts}
-                    />
-                  )}
-                </TabPanel>
+                <Tab>
+                  <TabText>Details</TabText>
+                </Tab>
                 {isSoftwareEnabled && (
-                  <TabPanel>
-                    <SoftwareCard
-                      id={deviceAuthToken}
-                      softwareUpdatedAt={host.software_updated_at}
-                      router={router}
-                      pathname={location.pathname}
-                      queryParams={parseHostSoftwareQueryParams(location.query)}
-                      isMyDevicePage
-                      platform={host.platform}
-                      hostTeamId={host.team_id || 0}
-                      isSoftwareEnabled={isSoftwareEnabled}
-                      onShowInventoryVersions={setHostSWForInventoryVersions}
-                    />
-                  </TabPanel>
+                  <Tab>
+                    <TabText>Software</TabText>
+                  </Tab>
                 )}
                 {isPremiumTier && (
-                  <TabPanel>
-                    <PoliciesCard
-                      policies={host?.policies || []}
-                      isLoading={isLoadingHost}
-                      deviceUser
-                      togglePolicyDetailsModal={togglePolicyDetailsModal}
-                      hostPlatform={host?.platform || ""}
-                      router={router}
-                    />
-                  </TabPanel>
+                  <Tab>
+                    <TabText count={failingPoliciesCount} countVariant="alert">
+                      Policies
+                    </TabText>
+                  </Tab>
                 )}
-              </Tabs>
-            </TabNav>
-            {showInfoModal && <InfoModal onCancel={toggleInfoModal} />}
```

Comment:
Just curious what the impact of moving this is? Just easier to find in the markup?

Reply:
the child is not necessarily rendered